### ISSUE TITLE
Mixing `@delete` and `authorize`?

### ISSUE THREAD

**Issue by robdmoore** [algorand-devrel/beaker#45]
H,

Loving using this library so far. Is it possible to mix `authorize` with `@delete`? I couldn't figure out how.

In the end I just did this:

```python
    @delete
    def delete():
        return Cond([Txn.sender() == Global.creator_address(), Approve()])
```

**Comment by barnjamin** [comment]
It _should_ be possible, I'll give it a shot. Thanks for the idea!

**Comment by barnjamin** [comment]
merged and pushed a tag 0.0.9-dev to pip
